### PR TITLE
Customers, Funding, Partnerships - Tandem

### PR DESCRIPTION
**Pull request by rajalkothari**

### CODE REVIEW

File: data/account-providers/tandem-money-limited.json [hunk 1]
```diff
@@ -74,11 +74,11 @@
   "github": null,
   "fca": "001b000000i8ezfAAA",
   "articles": [],
-  "thirdPartyBankingLicense": null,
+  "thirdPartyBankingLicense": false,
   "strategies": [],
-  "prepaidCards": [],
-  "debitCards": [],
-  "creditCards": [],
+  "prepaidCards": false,
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:79]: Card data points should always be arrays []. Same when empty

File: data/account-providers/tandem-money-limited.json [hunk 2]
```diff
@@ -74,11 +74,11 @@
   "github": null,
   "fca": "001b000000i8ezfAAA",
   "articles": [],
-  "thirdPartyBankingLicense": null,
+  "thirdPartyBankingLicense": false,
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:77]: You can't use a boolean for this data point. Should be an empty string "" or null.

File: data/account-providers/tandem-money-limited.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
   ],
   "apiStandards": [],
   "webApplication": false,
-  "mobileApps": [],
+  "mobileApps":true,
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:68]: You can't use a boolean for this data point. Follow this format https://github.com/not-a-bank/open-banking-tracker-data/blob/21242c234a16048a37b4807e3c6c2099c52967c4/data/account-providers/revolut.json#L116

File: data/account-providers/tandem-money-limited.json [hunk 4]
```diff
@@ -95,5 +95,45 @@
...
+        "amount": "500000",
+        "type": "claim",
+        "product": "currentAccounts",
+        "source": {
+          "url": "https://www.finextra.com/pressarticle/76995/tandem-reaches-half-a-million-customers",
+          "date": "2019-01-18",
+          "verified": true
+        }
+      }, 
+    ],
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:111]: Wrong closing of the customers object

File: data/account-providers/tandem-money-limited.json [hunk 5]
```diff
@@ -95,5 +95,45 @@
...
+  "fundingTotal": {
+    "amount": "134300000",
+    "currency": "GBP"
+  },
+  "foundedYear": 2013,
+  "partnerships": [
+    {
+      "name": "Truelayer",
+      "icon": "https://pbs.twimg.com/profile_images/1224754362477613057/krEOXZSf_400x400.jpg",
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:120]: Update url to https://res.cloudinary.com/apideck/icons/truelayer.jpg

File: data/account-providers/tandem-money-limited.json [hunk 6]
```diff
@@ -95,5 +95,45 @@
...
.../EJxwJHOLKE+jZQpihSBAgAtQmVQ4YLY8AqSSH4SUYEJfJLFQP936CMwdLLIERxvOWb2V+TJhL6BBwIELwAvTwakkZcABYECHZATK9jhwoQYNU02sPKEqwB04uPEhVFNHpxd9rUgErCSakKgo9dD7N5xx0bFRITfHavlV+y0orSDW/OM06bSOnHmhuoUPqqoh10uEqJQOVpAdIRZG8zKK2W8mUQUrSC0TUM4zlq3aNH4LmVl+QeoZxek7anCRB+5T+IKK2yKLFInaANEmvkEu3uI3PklvINsVifaeinJtRIld2P0z/kIpC3JcmvTLzNIfUlH2DosSl9hDGiKc+3YcF734AMIQ3/BS5p6xJMCRE5B5pRiG5siT74SA8t2Zn1ogipXJyDQk5W/XDDtFkghHVexJDTi1+fYi4zDyuHARZ6Qy5fcrxseCKGcbV79sNtvxK8eeF0WC7dr+zSKradcXDnRBl+WGnHq9OLuF3+oUo7xAPuEXhxeerp9l2XTKby89DknauvGhMAQ4ghKy/GE+YHtce44SQylOHc8JFpr74XTU8I30NkZCQe7fhw2hzWBflleOW+arJVcmZAIYCg/DNdwML6rf703Q3a3gnYj8gBOfXLq5ZCj4UhFPQ48NP2UAKAoOQAvR851PWA0LUG9rmiSlkZsEgNvDZ4YesQAUBVuTbRatdgmI4CJeIixas1YEQ0f25zudACwKCkBJv0ToCHTAI3qI1cweFnLEcwQnqAEHsti/L3g14YlUQ3jmEAL+e5neSap8PDQQiUX4HvL6gssEOCMQuUb5dwNqxMdwBg6D8MnvCEOoBm1o8RzizbziKAwLhZEzyBFaV6drGcYAgF0pxBOxTknIMX68wIFXssVyn3bYqyzXBaPNrwJC6LHaZ7VpblmeJVg6OAqkpY1Ie2ry2EaRnhslAgtSxx0Tkx7QJJoiOBcdlIEJ4bRnbND9slve/kNUKZ5tzKcc2MgMXUoXUGEKL82GTRrP/fO1GEMZRsjnkpbRt7GJcggypQ0jVM2ObSybKosjz/JTnRUmZvJSM9KGoYpHeYlcZ31L/ob9DVekR8v/GQHSLgegWA9EtBqJbDES3GIhueRyI/SAxEN1iILrFQHSLgegWA9EtBqJbHgcyfpAYiG4xEN1iILrFQHSLgegWA9EtDwP5F3srxV7JxfrIAAAAAElFTkSuQmCC",
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:130]: update url to https://res.cloudinary.com/apideck/icons/stripe

File: data/account-providers/tandem-money-limited.json [hunk 7]
```diff
@@ -95,5 +95,45 @@
...
+      "websiteUrl": "https://stripe.com/en-be",
+      "id": "stripe-usa",
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:132]: change id to stripe if the main domain is a .com

File: data/account-providers/tandem-money-limited.json [hunk 8]
```diff
@@ -95,5 +95,45 @@
...
+    "amount": "134300000",
+    "currency": "GBP"
+  },
+  "foundedYear": 2013,
+  "partnerships": [
+    {
+      "name": "Truelayer",
+      "icon": "https://pbs.twimg.com/profile_images/1224754362477613057/krEOXZSf_400x400.jpg",
+      "websiteUrl": "https://truelayer.com/",
+      "id": "truelayer-uk",
```
Comment (Gdewilde) [inline data/account-providers/tandem-money-limited.json:122]: change id to truelayer

**Comment by rajalkothari** [comment]
File amended - Tandem

**Pull request merged**